gateapi/pipeline_templates_controller_api.go

```diff
@@ -493,19 +500,24 @@ func (a *PipelineTemplatesControllerApiService) UpdateUsingPOST(ctx context.Cont
 	localVarPostBody = &pipelineTemplate
 	r, err := a.client.prepareRequest(ctx, localVarPath, localVarHttpMethod, localVarPostBody, localVarHeaderParams, localVarQueryParams, localVarFormParams, localVarFileName, localVarFileBytes)
 	if err != nil {
-		return nil, err
+		return successPayload, nil, err
 	}
 
 	localVarHttpResponse, err := a.client.callAPI(r)
 	if err != nil || localVarHttpResponse == nil {
-		return localVarHttpResponse, err
+		return successPayload, localVarHttpResponse, err
 	}
 	defer localVarHttpResponse.Body.Close()
 	if localVarHttpResponse.StatusCode >= 300 {
 		bodyBytes, _ := ioutil.ReadAll(localVarHttpResponse.Body)
-		return localVarHttpResponse, reportError("Status: %v, Body: %s", localVarHttpResponse.Status, bodyBytes)
+		return successPayload, localVarHttpResponse, reportError("Status: %v, Body: %s", localVarHttpResponse.Status, bodyBytes)
 	}
 
-	return localVarHttpResponse, err
+	if err = json.NewDecoder(localVarHttpResponse.Body).Decode(&successPayload); err != nil {
+		return successPayload, localVarHttpResponse, err
+	}
+
+
+	return successPayload, localVarHttpResponse, err
```

Comment:
Are there no changes necessary in the commands to output the reference to the Orca op?

Reply:
Ah I see, I meant from `spin`'s commands, e.g. [pipeline-templates save command](https://github.com/spinnaker/spin/blob/master/cmd/pipeline-template/save.go#L107). Are you consuming the [gateapi](https://github.com/spinnaker/spin/tree/master/gateapi) without using `spin`?

Reply:
Yes, I'm using it as a library. The `spin` command uses the V2 API for this so should be unaffected by these changes.